modulefiles/modulefile.ProdGSI.jet

```diff
@@ -11,21 +11,23 @@
 #module-whatis " GDAS_ENKF whatis description"
 #
 #
-module load newdefaults
-module load intel/2016.2.181 
-module load impi/5.1.1.109
+
+module load intel/18.0.5.274
+module load impi
 #
-module load szip/2.1   
+module load szip/2.1
 module load hdf5/1.8.9
-module load netcdf4/4.2.1.1  
+module load netcdf4/4.2.1.1
 
-export COREPATH=/lfs3/projects/hfv3gfs/nwprod/lib
-export COREPATH=/mnt/lfs3/projects/hfv3gfs/gwv/ljtjet/lib
+#export COREPATH=/lfs3/projects/hfv3gfs/nwprod/lib
+export COREPATH=/lfs4/HFIP/hfv3gfs/gwv/l0530/lib
 export NCEPLIBS=$COREPATH
 module use $NCEPLIBS/modulefiles
 
-export FFLAGS="-openmp"
-export LDFLAGS="-openmp"
+#export FFLAGS="-openmp"
```

Comment:
Remove all these commented lines from the modulefile for Jet.

Reply:
Removed.